about.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+}
+
+body {
+  background-color: #789677;
+  text-align: center;
+}
+
+h1 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 45px;
+  padding: 40px;
+}
+
+.center img {
+  position: relative;
+  height: 400px;
+  width: 40vw; 
```

Comment:
for mobile (max-width: 699px), it would look better if you took this width off. just do width: 100%; or width: auto;